Formula/mariadb.rb

```diff
@@ -216,17 +216,3 @@ def plist
 end
 
 __END__
```

Comment:
```suggestion
```
No, you need to remove this. Is the patch no longer needed?

Reply:
Sorry, I don’t know if it’s needed or not.